src/easy.rs

```diff
@@ -2864,6 +2958,11 @@ impl Easy {
         }
     }
 
+    fn getopt_double(&mut self, opt: curl_sys::CURLINFO) -> Result<c_double, Error> {
+        self.getopt_ptr(opt)
+            .map(|p| unsafe {transmute::<_, c_double>(p as u64)})
```

Comment:
I think we'll still want to call the raw FFI function directly here b/c we want 64-bits of information back, but on 32-bit this is only giving us 32-bits of information back